### PR TITLE
fix(releases): Scheduling info capitalization

### PR DESCRIPTION
**Pull request by madhurisandbhor**
### What does it do?

- It fixes the issues of capitalising the first letter of Hours/Days/Weeks etc.
- StyledComponents dependency upgraded to fix : defaultProps circularly references error.

### How to test it?
Please check releases page and check if scheduling info, eg: In 2 days, 6 hours, days/hours/week should not be capitalized.

### CODE REVIEW

File: packages/core/content-releases/package.json [hunk 1]
```diff
@@ -78,7 +78,7 @@
     "@testing-library/react": "14.0.0",
     "@testing-library/user-event": "14.4.3",
     "@types/koa": "2.13.4",
-    "@types/styled-components": "5.1.26",
+    "@types/styled-components": "5.1.32",
```
Comment (joshuaellis) [inline packages/core/content-releases/package.json:81]: Can you do this change everywhere please? We try to keep these deps in sync across the app.

File: packages/core/content-releases/admin/src/pages/ReleasesPage.tsx [hunk 2]
```diff
@@ -63,8 +63,10 @@ const LinkCard = styled(Link)`
   display: block;
 `;
 
-const CapitalizeRelativeTime = styled(RelativeTime)`
-  text-transform: capitalize;
+const FirstLetterStyle = styled(Box)`
```
Comment (joshuaellis) [inline packages/core/content-releases/admin/src/pages/ReleasesPage.tsx:66]: Does this actually need its own component? How come? 🤔

Comment (madhurisandbhor) [reply packages/core/content-releases/admin/src/pages/ReleasesPage.tsx:66]: its works only on block elements

Comment (joshuaellis) [reply packages/core/content-releases/admin/src/pages/ReleasesPage.tsx]: TIL. can it be `inline-block` and does that break anything? 🤔

Comment (madhurisandbhor) [reply packages/core/content-releases/admin/src/pages/ReleasesPage.tsx]: yeah why not, I am so in hurry 😖

**Pull request merged**